test/cpp/jit/test_schema_info.cpp

```diff
@@ -173,6 +173,48 @@ TEST(SchemaInfoMayAliasTest, AliasingInputOutput) {
       {c10::SchemaArgType::input, 1}, {c10::SchemaArgType::output, 0}));
 }
 
+TEST(SchemaInfoMayAliasTest, MultipleWildcardInputs) {
+  SchemaInfo schema(
+      "aten::test.Tensor(Tensor(a) a, Tensor(*) b, Tensor(*) c) -> (Tensor(a), Tensor(*))");
+  ASSERT_TRUE(schema.may_alias(
+      {c10::SchemaArgType::input, 0}, {c10::SchemaArgType::output, 0}));
```

Comment:
a `namespace ArgType = c10::SchemaArgType;` declaration would make ti so that you could write this test much more readable.